assets/src/application/ui/input/InlineEditInput/InlineEditText.tsx

```diff
@@ -1,10 +1,10 @@
 import React from 'react';
-import { TextProps } from 'antd/es/typography/Text';
+import { TextProps } from './types';
 
 import Editable from './Editable';
 
-const InlineEditText: React.FC<TextProps> = ({ ...rest }) => {
-	return <Editable inputType='text' ellipsis {...rest} />;
+const InlineEditText: React.FC<TextProps> = ({ ellipsis = true, ...rest }) => {
+	return <Editable inputType='text' ellipsis={ellipsis} {...rest} />;
```

Comment:
`ellipsis` could still be overridden via `...rest`

Reply:
oh right... i forget that prop order === precedence